KTXContainerFormat.md

```diff
@@ -0,0 +1,17 @@
+# KTX Container Format
+
+When implementing support for compressed textures, developers should be aware of three conceptual formats:
```

Comment:
Consider reordering the list below to make it aligned with how data is processed on the client: container-transmission-gpu.